python-checks/src/test/resources/checks/genericTypeWithoutArgumentImporting.py

```diff
@@ -1,5 +1,11 @@
-# mod.py
-from genericTypeWithoutArgumentImported import SomeGeneric, SomeGenericWithTypeParam, MyImportedChild, MyImportedChild2, MyImportedChild3
+from genericTypeWithoutArgumentImported import (
+    SomeGeneric,
+    SomeGenericWithTypeParam,
+    MyImportedGenericTypeVarChild,
+    MyImportedNonGenericChild,
+    MyImportedConcreteChild,
+    SomeGenericIncorrectlyDefined
+)
```

Comment:
may be?
```suggestion
from genericTypeWithoutArgumentImported import *
```

Reply:
Since support of wildcard import is brittle at the moment, I don't want to add confusion to this test: I prefer to keep the explicit import.